app/controllers/step/archives_controller.rb

```diff
@@ -20,6 +20,7 @@ def create
           end
         else
           format.html do
+            flash.now[:alert] = error_messages(@step.errors)
```

Comment:
Instead of using `:error_messages` which uses `:full_messages` (which smooshes the error name/tag together with the error message in a weird way -- I refer to this in the comment on concerns/workflow_metadata.rb), can we maybe do: 

`@step.errors.messages.values.flatten.map(&:capitalize).join("; ")` 

Or put that logic in a new `:error_messages_without_errornames` (or something) method in application_controller to reuse it? 

Or redefine `:error_messages` as:

```
  # @param errors [Errors]
  # @param mode [nil, :message_only]
  def error_messages(errors, mode = nil)
    case mode
    when :message_only
      errors.messages.values.flatten.map(&:capitalize).join('; ')
    else
      errors.full_messages.map { |msg| msg }.join('; ')
    end
  end
``` 

and pass the mode :message_only here and in the other step controllers.